packages/ui5-library-writer/templates/optional/typescript/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+      "target": "es2015",
+      "module": "es2015",
+      "moduleResolution": "node",
+      "skipLibCheck": true,
+      "preserveConstEnums": true,
+      "sourceMap": false,
+      "allowJs": true,
+      "strict": true,
+      "strictNullChecks": true,
+      "strictPropertyInitialization": true,
+      "rootDir": "./",
+      "outDir": "./src-gen",
+      "baseUrl": "./",
+      "typeRoots": [
+          "./node_modules/@types",
+          "./node_modules/<%= tsTypes %>"
+      ],
+      "paths": {
+          "<%= libraryNamespaceURI %>/*": [
+              "./src/<%= libraryNamespaceURI %>/*"
+          ]
+      }
+  },
+  "include": [
+      "./src/**/*",
+      "./test/**/*"
+  ]
+}
```

Comment:
Should be aligned with https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application-writer/templates/optional/typescript/tsconfig.json with some lib specific differences.

Reply:
Updated, please advise if further changes required

Reply:
Updated, please advise if further changes required